static/app/components/collapsible.tsx

```diff
@@ -13,6 +13,7 @@ type ExpandButtonRenderProps = {
 };
 
 type Props = {
+  children?: React.ReactNode;
```

Comment:
in a component above you define it as Non-optional and here as optional...is this on purpose?

Reply:
oop, I misread it, it does always need children, I'll update!